pkg/util/mount/nsenter_mount.go

```diff
@@ -170,6 +170,12 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, err
 	}
 
+	// Check the directory exists
+	if _, err = os.Stat(file); os.IsNotExist(err) {
```

Comment:
I agree. The change only affects the situation when the directory does not exist. I just want to make sure we are clear about semantics.